src/Commands/TestMakeCommand.php

```diff
@@ -18,7 +19,10 @@ class TestMakeCommand extends GeneratorCommand
 
     public function getDefaultNamespace() : string
     {
-        return $this->laravel['modules']->config('paths.generator.test.path', 'Tests');
+        if ($this->option('unit')) {
```

Comment:
This looks like unit is not default yet, shouldn't this be changed to feature too?